GettingStarted.md

```diff
@@ -1,57 +1,55 @@
 This document describes how to prepare the HoloToolkit sourcecode for use
 in your project.
 
-# Preparing the HoloToolkit-Unity Code
+# 1. Download a HoloToolkit-Unity Asset Package
```

Comment:
I'm not sure this is as much of a 1, 2, 3, but more of a 1a, 1b perhaps. Or just some presentation of the two options instead of one being "Step 1, now skip to step 4".